src/blocks/events/edit.js

```diff
@@ -50,58 +48,11 @@ const EventsEdit = ( props ) => {
 		innerBlocks: select( 'core/block-editor' ).getBlocks( clientId ),
 	} ) );
 
-	let observer = useRef( null );
-	let elementRef = useRef( null );
-	let slickTarget = useRef( null );
-
 	const externalCalendarUrl = attributes.externalCalendarUrl;
 	const [ isEditing, setIsEditing ] = useState( false );
 	const [ showExternalCalendarControls, setShowExternalCalendarControls ] = useState( !! externalCalendarUrl || false );
 	const [ stateExternalCalendarUrl, setStateExternalCalendarUrl ] = useState( externalCalendarUrl );
 
-	useEffect( () => {
```

Comment:
Oh! It's not needed anymore?

Reply:
Closing this PR as we will need to refactor the Events block to use Tiny Swiper instead of Slick